src/database/mixxxdb.h

```diff
@@ -19,8 +19,8 @@ class MixxxDb : public QObject {
 
     static bool initDatabaseSchema(
             const QSqlDatabase& database,
-            const QString& schemaFile = kDefaultSchemaFile,
-            int schemaVersion = kRequiredSchemaVersion);
+            int schemaVersion = kRequiredSchemaVersion,
```

Comment:
I do not agree to include unrelated and unnecessary changes in this PR.

Reply:
> This is just an assumption and guessing.

Nope, I actually wanted to change the target version in a test - but I didn't want to change the file, so I would have had to specify the path again.
Here's the thing: If you supply a different schema file, versions don't mean the same thing, so using the default required version wouldn't make sense. On the other hand, targeting a different version in the same file is a totally valid scenario.

Reply:
Ok. This information was missing.